profiles/models.py

```diff
@@ -88,21 +91,23 @@ class Profile(models.Model):
     founded = models.SmallIntegerField(
         validators=[validate_foundation_year_range], default=None, null=True
     )
-    service_info = models.TextField(default=None, null=True)
-    product_info = models.TextField(default=None, null=True)
-    address = models.TextField(default=None, null=True)
-    startup_idea = models.TextField(default=None, null=True)
+    service_info = models.TextField(blank=True, default="")
+    product_info = models.TextField(blank=True, default="")
+    address = models.TextField(blank=True, default="")
+    startup_idea = models.TextField(blank=True, default="")
 
     banner_image = models.ImageField(
         upload_to="banners",
         validators=[validate_image_format, validate_image_size],
         null=True,
+        blank=True,
     )
 
     logo_image = models.ImageField(
         upload_to="logos",
         validators=[validate_image_format, validate_logo_size],
         null=True,
+        blank=True,
     )
```

Comment:
God help you, guys, with managing `ImageField` (or any other file field) on model which is usually served by API.

Sorry for missing it when it was implemented. Just keep in mind, in general, that if you have a feeling that it was a bad idea — it really was. :)

Better approach: to have a separate model _specifically_ for images (and/or other files). To use a separate endpoints for files upload. And to have FKs to that model for files handling in the core models.